docs/docs/platform/service-replicas.mdx

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
maybe a note about pricing, that this is only supported for people with dedicated resources and that each replica adds to the total price by adding the compute

Reply:
done.